x-pack/spec/filters/geoip/database_manager_spec.rb

```diff
@@ -357,8 +361,8 @@ def expect_healthy_database_metric(c)
         end
 
         it "should return nil" do
-          allow(mock_download_manager).to receive(:fetch_database).and_raise("boom")
```

Comment:
mock_download_manager is not triggered 
`db_manager.subscribe_database_path()` use the real `download_manager`